src/include/packet_types.h

```diff
@@ -155,4 +155,12 @@ typedef struct altitude_data_block {
 
 void altitude_data_block_init(AltitudeDataBlock *b, const int8_t measurment_type, const uint8_t pressure,
                               const uint8_t temperature, const uint8_t altitude);
-#endif // _PACKET_TYPES_H
+
+typedef struct angular_velocity_block {
+    /**The angular velocity block accessef as a bytes array*/
+    uint8_t bytes[12];
+} TIGHTLY_PACKED AngularVelocityBlock;
+
+void angular_velocity_block(AngularVelocityBlock *b, const int8_t measurment_time, const int8_t full_scale_range,
```

Comment:
`full_scale_range` and `measurement_time` should be unsigned values!

Reply:
Fixed!